docs/import/add_disaster.js

```diff
@@ -284,14 +284,12 @@ function createAssetPickers(states) {
 /**
  * Deletes a disaster from firestore. Confirms first. Returns when deletion is
  * complete (or instantly if deletion doesn't actually happen).
- * @param {Window} globalWindow DOM Window (for test injection)
  * @return {Promise<void>}
  */
-function deleteDisaster(globalWindow) {
+function deleteDisaster() {
```

Comment:
there's a ref to this method further up in enableWhenReady do you mind getting rid of the param there as well. Thanks!